packages/app/src/test-utils.tsx

```diff
@@ -15,17 +16,25 @@ interface RenderAppResult extends RenderResult {
 type InitialPath = `/${string}`;
 interface RenderAppOpts {
   initialPath?: InitialPath;
+  preferredVis?: Vis | undefined;
 }
 
 export async function renderApp(
   opts: InitialPath | RenderAppOpts = '/'
 ): Promise<RenderAppResult> {
   const optsObj = typeof opts === 'string' ? { initialPath: opts } : opts;
-  const { initialPath }: RenderAppOpts = {
+  const { initialPath, preferredVis }: RenderAppOpts = {
     initialPath: '/',
     ...optsObj,
   };
 
+  if (preferredVis) {
+    window.localStorage.setItem(
+      'h5web:preferredVis',
+      JSON.stringify(preferredVis)
+    );
+  }
```

Comment:
I know we clear the `localStorage` after each test but should we need to clear it aso **before** if we start relying on this setting ?

Reply:
Doing both is not needed, since when Jest starts, it creates a fresh jsdom environment, so there's never anything in `localStorage`. We could clear it _before_ instead of after each test, but conceptually, it feels like a clean up step rather than a preparation step.

Reply:
All right :+1: